admin/controllers/school.js

```diff
@@ -23,7 +20,7 @@ const getHome = async (req, res, next) => {
   let schoolName = ''
 
   try {
-    const school = await School.findOne({ '_id': req.user.School }).exec()
+    const school = await schoolDataService.findOne({ '_id': req.user.School })
```

Comment:
can we add a todo for future data layer extraction? it will be easier to track.

Reply:
Please explain?

Reply:
Whenever I come across dataservice changes in a controller i am simply adding a TODO above.
Example in school controller on my PR:
  `// TODO: extract this dataservice call to a service
  const latestCheck = await checkDataService.findLatestCheck({ pupilId: pupilId })`